deluge/ui/web/server.py

```diff
@@ -748,6 +831,14 @@ def start_normal(self):
         ip = self.socket.getHost().host
         ip = '[%s]' % ip if is_ipv6(ip) else ip
         log.info('Serving at http://%s:%s%s', ip, self.port, self.base)
+        if self.webapidoc_enabled:
```

Comment:
I'm not sure what value this log entry adds

Reply:
It shows where the docs are served. Isn't that useful in itself?